cookiecutter.json

```diff
@@ -20,5 +20,9 @@
     "use_compressor": "n",
     "js_task_runner": ["Gulp", "Grunt", "None"],
     "use_lets_encrypt": "n",
+    "use_gitlab_ci": "n",
+    "staging_branch": "",
+    "staging_domain_name": "beta.{{ cookiecutter.domain_name }}",
+    "production_branch": "",
```

Comment:
How about if the option for use_gitlab_ci was "No", "Test Only" or "Test and Deploy"?  If the user chooses "Test and Deploy" then the master branch is automatically deployed to {{ cookiecutter.domain_name }}.

That would make it easier to get started with continuous deployment.  Someone wanting a more complex deployment configuration (different branch, staging site, etc) can modify .gitlab-ci.yml to their needs after the project in generated.